### PR TITLE
Add task for removing users

### PR DESCRIPTION
**Pull request by danielitit**

### CODE REVIEW

File: molecule/add_user/tests/test_default.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import os
+
+import testinfra.utils.ansible_runner
+
+#testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+#    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
```
Comment (norberttech) [inline molecule/add_user/tests/test_default.py]: development leftover?

File: molecule/add_user/molecule.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    cpus: 1
+    privileged: true
+provisioner:
+  name: ansible
+  lint:
+    name: ansible-lint
+scenario:
+  name: add_user
+  test_sequence:
+    #  - lint
```
Comment (norberttech) [inline molecule/add_user/molecule.yml]: can we bring it back?

Comment (danielitit) [reply molecule/add_user/molecule.yml]: Yes. We can. Done

File: molecule/add_user/tests/test_default.py (hunk 1 above)
Comment (danielitit) [reply molecule/add_user/tests/test_default.py]: Yes. Solved.

**Pull request merged**